src/interfaces/restaurant.ts

```diff
@@ -61,7 +65,7 @@ export interface PlaceType {
 }
 
 export interface IRestaurantMainFilters {
-    geoCode: IGeolocation;
+    geoCode: IGeolocation | null | undefined;
```

Comment:
Creo que sería lo mismo que escribirlo así

```suggestion
    geoCode?: IGeolocation;
```